pkgs/applications/audio/non/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, pkgs, fetchgit}:
+
+stdenv.mkDerivation rec {
+  name = "non";
+  version = "2015-10-6";
+  src = fetchgit {
+    url = "git://github.com/original-male/non.git";
```

Comment:
If the repos are in sync, and the homepage refers to the "..../non.git" repo, why not use that?